lib/pages/settings_page.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:MobileOne/localization/localization.dart';
+import 'package:MobileOne/services/color_service.dart';
+import 'package:MobileOne/services/messaging_service.dart';
+import 'package:MobileOne/services/preferences_service.dart';
+import 'package:MobileOne/services/user_service.dart';
+import 'package:MobileOne/utility/colors.dart';
+import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
+
+class SettingsPage extends StatefulWidget {
+  SettingsPage({Key key}) : super(key: key);
+
+  @override
+  SettingsPageState createState() => SettingsPageState();
+}
+
+class SettingsPageState extends State<SettingsPage> {
+  final _colorsApp = GetIt.I.get<ColorService>();
+  final _sharedPreferences = GetIt.I.get<PreferencesService>();
+  final _messagingService = GetIt.I.get<MessagingService>();
+  final _userService = GetIt.I.get<UserService>();
+
+  bool onSharedWishlist = true;
+
+  @override
+  void initState() {
+    if (_sharedPreferences.getBool("onSharedWishlist") != null) {
+      onSharedWishlist = _sharedPreferences.getBool("onSharedWishlist");
+    }
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: _colorsApp.colorTheme,
+      body: SingleChildScrollView(
+        child: SafeArea(
+          child: Column(
+            children: <Widget>[
+              Align(
```

Comment:
stop making your own app bar. use the AppBar widget provided by flutter in the dedicated scaffold field